.github/workflows/main.yml

```diff
@@ -0,0 +1,62 @@
+# The name of the job, which will be displayed for the status check in PR.
+name: README and metadata formatting
+
+# Controls when the action will run.
+# Below triggers the workflow on pull requests to the specified branches.
+on:
+  pull_request:
+    branches:
+      - master
```

Comment:
Yeah correct - it would just be a case of making that manual update. All that would happen if the branch was renamed and we didn't add it, is the scripts just would not run on that branch, so that should help us remember / it wouldn't cause any direct issues.